enclave-runtime/src/beefy_merkle_tree.rs

```diff
@@ -0,0 +1,229 @@
+// This file is part of Substrate.
```

Comment:
This file was copy pasted from https://github.com/paritytech/substrate/blob/master/frame/beefy-mmr/primitives/src/lib.rs. Can be removed once we are above commit https://github.com/paritytech/substrate/commit/65ac6f39c72e77fd98f337f1a5beddd539ee8d6f